src/makielayout/blocks/axis.jl

```diff
@@ -327,6 +327,7 @@ function initialize_block!(ax::Axis; palette = nothing)
         ticklabelsize = ax.xticklabelsize, trimspine = ax.xtrimspine, ticksize = ax.xticksize,
         reversed = ax.xreversed, tickwidth = ax.xtickwidth, tickcolor = ax.xtickcolor,
         minorticksvisible = ax.xminorticksvisible, minortickalign = ax.xminortickalign, minorticksize = ax.xminorticksize, minortickwidth = ax.xminortickwidth, minortickcolor = ax.xminortickcolor, minorticks = ax.xminorticks, scale = ax.xscale,
+        minorticksused = ax.xminorgridvisible,
```

Comment:
What's the difference between `minorticksvisible = ax.xminorticksvisible` and `minorticksused = ax.xminorgridvisible` ?